src/styles/utils/_mediaqueries.scss

```diff
@@ -0,0 +1,11 @@
+@media (min-width:600px) {
+  .container {
+    grid-template-columns: 1fr 1fr;
+  }
+}
+
+@media (min-width:900px) {
+  .container {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
```

Comment:
move media queries to the container file, no need to store them in separate files
also use minmax function instead of min-width property on each box